deployments/ansible/roles/collector/README.md

```diff
@@ -110,10 +116,11 @@ how to use the role in a playbook with minimal required configuration:
   stretch, and `4.1.1` for other distros`)
 
 - `splunk_fluentd_config`: Path to the fluentd config file on the remote host.
-  (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
+  (**default:** `/etc/otel/collector/fluentd/fluent.conf` on Linux, 
+  **default:** `%SYSTEMDRIVE%\opt\td-agent\etc\td-agent\td-agent.conf` on Windows)
 
 - `splunk_fluentd_config_source`: Source path to a fluentd config file on your 
   control host that will be uploaded and set in place of `splunk_fluentd_config` on
   remote hosts. Can be used to submit a custom fluentd config,
-  e.g. `./custom_fluentd_config.conf`. (**default:** `""` meaning 
-  that nothing will be copied and existing `splunk_fluentd_config` will be used)
+  e.g. `./custom_fluentd_config.conf`. (**default:** `""` 
+  meaning that nothing will be copied and existing `splunk_fluentd_config` will be used)
```

Comment:
Please discard this change. Try to keep you diffs always related to the change you are working on